### PR TITLE
Make December 2023 news public

### PR DESCRIPTION
**Pull request by cticenhour**
Add January 2024 template

Refs #11447

@idaholab/moose-ccb The newsletter for December is completely empty. Please add your contributions!

### CODE REVIEW

File: modules/doc/content/newsletter/index.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ monthly to the [MOOSE discussion forum](contact_us.md) as well as provided below
 
 ## 2023
 
+- [December, 2023](2023_11.md)
```
Comment (cticenhour) [inline modules/doc/content/newsletter/index.md:9]:
```suggestion
- [December, 2023](2023_12.md)
```

File: modules/doc/content/newsletter/2023/2023_12.md [hunk 2]
```diff
@@ -1,14 +1,42 @@
...
+- spatially restrict both origin and target domains
+- match either origin or target divided regions to an (usually child) application
+- match mesh division regions with the same index in the source and target region
+
+
+### Distributing grid MeshDivisions using Positions
+
+The [CartesianGridDivision.md], [CylindricalGridDivision.md] and [SphericalGridDivision.md]
+can now be composed with a [Positions object](syntax/Positions/index.md) to distribute the centers
+of several grids on the Positions. This can be used to define nested lattices for example.
```
Comment (cticenhour) [inline modules/doc/content/newsletter/2023/2023_12.md:32]:
```suggestion
of several grids on the Positions. For example, this new capability can be used to define nested lattices.
```

File: modules/doc/content/newsletter/2023/2023_12.md [hunk 3]
```diff
@@ -1,14 +1,42 @@
...
+nodal patch recovery will be used to compute the nodal degrees of freedom.
+
+### Transfers leveraging MeshDivisions
+
+[MultiAppGeneralFieldTransfer.md] derived transfers can now use [MeshDivisions](syntax/MeshDivisions/index.md)
+to:
+
+- spatially restrict both origin and target domains
+- match either origin or target divided regions to an (usually child) application
+- match mesh division regions with the same index in the source and target region
```
Comment (cticenhour) [inline modules/doc/content/newsletter/2023/2023_12.md:25]:
```suggestion
- spatially restrict both origin and target domains,
- match either origin or target divided regions to an application (usually a child), and
- match mesh division regions with the same index in the source and target region.
```

File: modules/doc/content/newsletter/2023/2023_12.md [hunk 4]
```diff
@@ -1,14 +1,42 @@
...
-for a complete description of all MOOSE changes.
-!alert-end!
-
 ## MOOSE Improvements
 
+### Projected stateful material properties
+
+The [ProjectedStatefulMaterialStorageAction](/ProjectedStatefulMaterialStorageAction.md)
+action system has been added to MOOSE. The action allows users to specify material
+properties that will be (componentwise) projected a finite element function basis
```
Comment (cticenhour) [inline modules/doc/content/newsletter/2023/2023_12.md:9]:
```suggestion
properties that will be (componentwise) projected onto a finite element function basis
```

File: modules/doc/content/newsletter/2023/2023_12.md [hunk 5]
```diff
@@ -1,14 +1,42 @@
...
+to store the old and older state of the properties.
+
+This is an *alternative* way of keeping old material property state, which permits
+interpolation at new or changing quadrature point locations (encountered - for
+example - with mortar contact or adaptivity). Both nodal as well as elemental basis
+functions can be used for the projection. In the case of nodal (Lagrange) functions
+nodal patch recovery will be used to compute the nodal degrees of freedom.
```
Comment (cticenhour) [inline modules/doc/content/newsletter/2023/2023_12.md:16]:
```suggestion
This is an *alternative* way of keeping an old material property state, which permits
interpolation at new or changing quadrature point locations (encountered, for
example, with mortar contact or adaptivity). Both nodal as well as elemental basis
functions can be used for the projection. In the case of nodal (Lagrange) functions,
nodal patch recovery will be used to compute the nodal degrees of freedom.
```

File: framework/doc/content/source/actions/ProjectedStatefulMaterialStorageAction.md [hunk 6]
```diff
@@ -8,4 +8,34 @@ system which sets up all necessary objects to project material property
...
+A [ProjectedStatefulMaterialAux](/ProjectedStatefulMaterialAux.md) aux kernel of the appropriate
+type will be added for each scalar material property component (e.g. vector or
+tensor components) to perform an elemental projection of the property component.
+
+#### Nodal basis functions
+
+A [ProjectedStatefulMaterialNodalPatchRecovery](/ProjectedStatefulMaterialNodalPatchRecovery.md)
+user object of the appropriate type will be created for each material property. To prepare data
+required for nodal patch recovery of each scalar material property component (e.g. vector or
+tensor components).
```
Comment (cticenhour) [inline framework/doc/content/source/actions/ProjectedStatefulMaterialStorageAction.md:37]:
```suggestion
A [ProjectedStatefulMaterialNodalPatchRecovery](/ProjectedStatefulMaterialNodalPatchRecovery.md)
user object of the appropriate type will be created for each material property to prepare data
required for nodal patch recovery of each scalar material property component (e.g. vector or
tensor components).
```

**Review by cticenhour — COMMENTED** [review]
@dschwen most of these adjustments were in your additions, so please take a look to see if I maintained the meaning you were after.

File: modules/doc/content/newsletter/2023/2023_12.md (hunk 3 above)
Comment (cticenhour) [reply modules/doc/content/newsletter/2023/2023_12.md:25]: @GiudGiud

File: modules/doc/content/newsletter/2023/2023_12.md (hunk 2 above)
Comment (cticenhour) [reply modules/doc/content/newsletter/2023/2023_12.md:32]: @GiudGiud

File: modules/doc/content/newsletter/2023/2023_12.md (hunk 3 above)
Comment (GiudGiud) [reply modules/doc/content/newsletter/2023/2023_12.md:25]: Happy with everything you suggested

File: modules/doc/content/newsletter/2023/2023_12.md [hunk 7]
```diff
@@ -1,14 +1,42 @@
...
+### Distributing grid MeshDivisions using Positions
+
+The [CartesianGridDivision.md], [CylindricalGridDivision.md] and [SphericalGridDivision.md]
+can now be composed with a [Positions object](syntax/Positions/index.md) to distribute the centers
+of several grids on the Positions. For example, this new capability can be used to define nested lattices.
+
+
 ## libMesh-level Changes
 
 ## PETSc-level Changes
```
Comment (cticenhour) [inline modules/doc/content/newsletter/2023/2023_12.md:37]:
```suggestion
```

**Review by loganharbour — APPROVED** [review]

**Review by GiudGiud — APPROVED** [review]

**Pull request merged**